### DISCUSSION TITLE
Why aren't all packages linked to the global repository?

### DISCUSSION THREAD

**Discussion by 0xdeadc0d3** [pnpm/pnpm#4326] [Q&A] [answered]
Why aren't all packages linked to the global repository? For example eslint
```
pnpm dlx create-react-app test-react-app --template typescript
cd test-react-app
rmdir /s /q node_modules
pnpm install
```
![image](https://user-images.githubusercontent.com/85187695/153710611-ea3b5e6a-a040-4271-9921-15bcb2eb6cd3.png)
![image](https://user-images.githubusercontent.com/85187695/153710617-94cc7a2c-bff8-4ad0-9ad9-5fd92d70e95d.png)

**Comment by zkochan** [comment, marked as answer]
Those files from the second screenshot are hard links to the global content-addressable store.

![image](https://user-images.githubusercontent.com/1927579/153775380-4de83422-cc4d-43cb-a75a-c0b310d5afa0.png)

**Reply by BlackHole1** [reply]
Great architecture diagram, it would be better if it was on https://pnpm.io/symlinked-node-modules-structure
